docker/deployed/mainnet/api/config.json

```diff
@@ -107,6 +107,34 @@
             },
             "HashCalculationStep": 450
         },
+        {
+            "Name": "Arbitrum Nova Mainnet",
+            "ChainID": 42170,
+            "AllowTransactionRelay": false,
```

Comment:
This flag `AllowTransactionRelay` is being removed in the Drop JSON-RPC work. I think you can remove it, to avoid conflicts later